rnef.config.mjs

```diff
@@ -2,13 +2,15 @@ import {platformAndroid} from '@rnef/platform-android';
 import {platformIOS} from '@rnef/platform-ios';
 import {pluginMetro} from '@rnef/plugin-metro';
 
+const isHybrid = process.env.IS_HYBRID_APP === 'true';
+
 /** @type {import('@rnef/config').Config} */
 export default {
     remoteCacheProvider: null,
     bundler: pluginMetro(),
     platforms: {
-        ios: platformIOS(),
-        android: platformAndroid(),
+        ios: platformIOS({sourceDir: isHybrid ? './iOS' : './ios'}),
```

Comment:
Confirming this one but looks like there's a bug specific to iOS we need to fix upstream. Android works with this configuration and iOS should resolve sourcePaths the same way (which it currently does not, failing on pods installation).

Reply:
For context, this seem to be the last missing piece of the puzzle. Newdot iOS builds fine if we specify `./` in here.